### DISCUSSION TITLE
Is there any trick to know when a serise has been archived done?

### DISCUSSION THREAD

**Discussion by luxunxiansheng** [dcm4che/dcm4chee-arc-light#3920] [Q&A] [answered]
Hi, 
     We want to convert a  series  into NIfTI  during archiving.  But we don't know when to trigger this process since it is hard to know if  this series has been transfered done.  Is there any workaround on it ?

Best Regards
L.B.

**Comment by vrindanayak** [comment]
@luxunxiansheng : Not sure what exactly are you implying by `NIfTI`. However, archive supports notifying systems using [RAD-49 Instance Availability Notification](https://www.ihe.net/uploadedFiles/Documents/Radiology/IHE_RAD_TF_Vol2.pdf#page=410) / [RAD-132 HL7 Procedure Status Update messages](https://www.ihe.net/uploadedFiles/Documents/Radiology/IHE_RAD_Suppl_EBIW.pdf#page=71) whenever a study (/ series) has been received by archive.

**Reply by luxunxiansheng** [reply]
From RAD49, the notification seems instance level other than serise level if I unserstand correctly.  That is,  a notificaiton will  be sent as long as an instance is ready.  But some post-processing won't start until a whole series reaches. I need a clear signal  that the whole sereis is ready on archive then I can retrieve it.  is it possible?

**Reply by vrindanayak** [reply, marked as answer]
> From RAD49, the notification seems instance level other than serise level if I unserstand correctly. That is, a notificaiton will be sent as long as an instance is ready. But some post-processing won't start until a whole series reaches. I need a clear signal that the whole sereis is ready on archive then I can retrieve it. is it possible?

We normally recommend the export always on SERIES level. It is not clear what kind of notification you want to send out - HL7 / IAN? 
- Instance Availability Notifications (IAN) are sent out on receive of Study / MPPS
- HL7 Procedure Status Update notifications are also sent out on receive of Study / MPPS. Additionally, we have an alternative to send this also using exporters / export rules which can be configured to export notifications on SERIES level i.e. when whole sereis is received on archive
